.tx/config

```diff
@@ -1,14 +1,14 @@
 [main]
 host = https://www.transifex.com
 
-[edx-platform.edx-django-extensions]
-file_filter = edx-django-extensions/conf/locale/<lang>/LC_MESSAGES/django.po
-source_file = edx-django-extensions/conf/locale/en/LC_MESSAGES/django.po
+[edx-platform.edx-management-commands]
```

Comment:
@clintonb I don't think this repo needs this file to begin with. Maybe you can just remove the entire `.tx` directory?